_drafts/oryx/routing.md

```diff
@@ -0,0 +1,60 @@
+# Routing
+
+Routing is an essential feature for any web application, allowing users to navigate between different pages or components within the application. The `@spryker-oryx/router` package
+provides routing infrastructure for your application, including a `RouterService` for interacting with the router and framework-specific integrations for providing actual routing config.
```

Comment:
Please split the last sentence into two. The last part, in particular, doesn't make sense for me